scripts/build-package

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+pip install --require-hashes -r requirements.txt --target ./tmp_package
+pip install --no-deps --target ./tmp_package .
+cp ./configuration/exodus-lambda-deploy.yaml ./tmp_package
+envsubst < ./configuration/lambda_config.json > ./tmp_package/lambda_config.json
+cd ./tmp_package
+ls
+aws cloudformation package \
+	--template exodus-lambda-deploy.yaml \
+	--s3-bucket codepipeline-us-east-1-956509269028 \
+	--output-template-file exodus-lambda-pkg.yaml
```

Comment:
Just a reminder, s3-bucket should point to "exodus-pipeline-artifacts" before merge